GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m

```diff
@@ -314,4 +359,44 @@ + (void)getGooglePlacesForSearchTerm:(NSString *)searchTerm
     }] resume];
 }
 
+#pragma mark - Debug Log
+
++(void)debugLogRequest:(NSURLRequest *)request {
+    NSString *size = [[NSByteCountFormatter new] stringFromByteCount:(long long)request.HTTPBody.length];
```

Comment:
Can we add a property on the plugin, `isDebuggingEnabled` ? Otherwise this might spam

Reply:
```
@property (nonatomic, getter = isDebuggingEnabled) BOOL debuggingEnabled;
```

Reply:
Done.